src/vs/workbench/contrib/chat/browser/contrib/editorHoverWrapper.ts

```diff
@@ -0,0 +1,52 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import 'vs/css!./media/editorHoverWrapper';
+import * as dom from 'vs/base/browser/dom';
+import { IHoverAction } from 'vs/base/browser/ui/hover/hover';
+import { HoverAction } from 'vs/base/browser/ui/hover/hoverWidget';
+import { IKeybindingService } from 'vs/platform/keybinding/common/keybinding';
+
+const $ = dom.$;
+const h = dom.h;
+
+/**
+ * This implements some of HoverWidget so that a chat editor hover can be rendered in the same way as a workbench hover.
+ * Maybe it can be reusable in a generic way.
+ */
+export class ChatEditorHoverWrapper {
```

Comment:
FYI @benibenj @Tyriar I had to do this to be able to render an editor hover the same way as the IHoverService hover. We could have HoverWidget also use this, but this is also a niche use-case and maybe nobody else will need to do what I'm doing here.

Reply:
Well, we might stop using the 'actions' for the chat hover anyway, so maybe then this just exists to add padding...